### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -89,7 +89,7 @@ There are two date arithmetic traps in the Gregorian calendar:
 
 In both cases, the result must be changed to the first day of the next month.
 
-Provided periods are represented by [start date inclusive, end date exclusive),
+Provided periods are represented by (start date inclusive, end date exclusive),
```
Comment (aaugustin) [inline README.rst:92]: This is the correct and intended mathematical notation. English uses [0, 1) where French uses [0, 1[.

Comment (aaugustin) [inline README.rst:92]: Upon further thought, it's likely going to confuse a large fraction of the intended audience, so I'll make the change anyway.

**Comment by aaugustin** [comment]
Merged.

File: README.rst (hunk 1 above)
Comment (davidbgk) [inline README.rst:92]: Oh I learned something, thanks!

**Pull request closed without merging**